packages/cursorless-org-docs/src/docs/contributing/CONTRIBUTING.mdx

```diff
@@ -138,11 +150,14 @@ Once you have your package then you can install it into the sandbox using the fo
 
 <Tabs groupId="vsx">
   <TabItem value="vscode" label="VSCode">
-    `bash code --profile=cursorlessDevelopment --install-extension bundle.vsix`
+    ```bash
+    code --profile=cursorlessDevelopment --install-extension bundle.vsix
+    ```
   </TabItem>
   <TabItem value="vscodium" label="VSCodium">
-    `bash codium --profile=cursorlessDevelopment --install-extension
-    bundle.vsix`
+    ```bash
```

Comment:
I suspect if we put blank lines around the code blocks prettier will have a better time

Reply:
Thank you! That worked.